keps/sig-node/1029-ephemeral-storage-quotas/README.md

```diff
@@ -576,6 +579,79 @@ required elsewhere:
   future allow adding additional data without having to change code
   other than that which uses the new information.
 
+#### Implementation details of using User Namespace to Toggle Ephemeral-Storage-Quotas
+
+* Without user namespaces, privileged users can alter projectIDs 
+on filesystems using tools like xfs_io, which can bypass the 
+controls established by filesystem project quotas designed to 
+monitor and optionally enforce resource consumption limits. 
+User namespaces restrict this, preventing users from making 
+changes to projectIDs outside their namespace. 
+
+* The implementation centers around a feature toggle which 
+controls the integration of xfs-quota with user namespaces for 
+managing filesystem quotas within containers. The toggle allows 
+to switch between a quota management system and a traditional 
+disk usage monitoring method.
```

Comment:
Can you describe where the toggle is and the entity responsible for setting/unsetting it?